asistente_ladm_col/gui/qgis_model_baker/dlg_import_schema.py

```diff
@@ -278,41 +265,29 @@ def accepted(self):
             self.import_models_list_widget.setFocus()
             return
 
-        self.save_configuration(configuration)
+        self.save_configuration(None)
```

Comment:
Couldn't you better remove the parameter in `save_configuration`. Seems to be useless there.